batch/batch/worker.py

```diff
@@ -58,6 +58,31 @@
 
 docker = aiodocker.Docker()
 
+port_allocator = None
+
+worker = None
+
+
+class PortAllocator:
+    def __init__(self):
+        # random unallocated ports take from
+        # https://stackoverflow.com/questions/10476987/best-tcp-port-number-range-for-internal-applications
```

Comment:
I don't see how this SO post is relevant to the choice of the range 46572-46582? Is 46572 just a random choice from one of the unassigned ranges in the second answer?

Reply:
Yes.  I updated the link to point to the second answer.